src/frontend/src/pages/LandingPage/LandingPage.scss

```diff
@@ -46,10 +54,17 @@
 }
 
 .info-section {
-  display: flex;
-  justify-content: space-evenly;
-  align-items: space-evenly;
-  height: 710px;
+  @include breakpoint(0 map-get($grid-breakpoints, 'xl')) {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+    display: flex;
+    justify-content: space-between;
+    align-items: space-evenly;
+    padding-bottom: calc($padding-block + $footer-height);
+    //height: 710px;
```

Comment:
commented code to be removed - or please add additional comments to justify the comment